api/tests/core/mails/transactional/bookings/beneficiary_booking_cancellation_test.py

```diff
@@ -0,0 +1,241 @@
+from datetime import datetime
+from datetime import timedelta
+
+from freezegun import freeze_time
+import pytest
+
+from pcapi.core.bookings import factories as booking_factories
+import pcapi.core.mails.testing as mails_testing
+from pcapi.core.mails.transactional.bookings.beneficiary_booking_cancellation import (
+    get_beneficiary_booking_cancellation_email_data,
+)
+from pcapi.core.mails.transactional.bookings.beneficiary_booking_cancellation import (
+    send_beneficiary_booking_cancellation_email,
+)
+from pcapi.core.offers.factories import EventStockFactory
+from pcapi.core.offers.factories import ThingStockFactory
+from pcapi.core.testing import override_features
+from pcapi.core.users.factories import BeneficiaryGrant18Factory
+
+
+@pytest.mark.usefixtures("db_session")
+class SendMailjetBeneficiaryBookingCancellationEmailTest:
+    @override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=False)
+    def test_should_called_send_email_with_valid_data(self):
+        # given
+        booking = booking_factories.IndividualBookingFactory()
+
+        # when
+        send_beneficiary_booking_cancellation_email(booking.individualBooking)
+
+        # then
+        assert len(mails_testing.outbox) == 1  # test number of emails sent
+        assert mails_testing.outbox[0].sent_data["Mj-TemplateID"] == 1091464
+
+
+@pytest.mark.usefixtures("db_session")
+class MakeBeneficiaryBookingCancellationEmailMailjetDataTest:
+    @override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=False)
```

Comment:
Tu dois pouvoir mettre `@override_features(ENABLE_SENDINBLUE_TRANSACTIONAL_EMAILS=False)` au niveau de la classe, pour ne pas avoir besoin de décorer chaque méthode

Reply:
E   TypeError: override_features cannot be used to decorate a class